.gitignore

```diff
@@ -35,3 +35,4 @@ buildNumber.properties
 
 .idea
 *.iml
+*.DS_Store
```

Comment:
Why do we need this change?

Reply:
I notice that when running `git add -A` command, some DS_Store files generated by Mac will be added too. I believe we do not want these files and adding it to gitigore helps developers. Do you have other concerns?

Reply:
I am just curious how the `DS_Store` files are generated.

Reply:
Typically, it is because I opened some files(e.g. the CI test report) of this github project with `Finder` in my local machine.

Reply:
nit: Keep the empty line at end of file.